packages/settings-library/src/settings_library/twilio.py

```diff
@@ -0,0 +1,23 @@
+""" Twilio settings
+
+"""
+
+from pydantic import Field
+
+from .base import BaseCustomSettings
+
+
+class TwilioSettings(BaseCustomSettings):
+    # SEE https://www.twilio.com/docs/sms/quickstart/python
+    TWILIO_ACCOUNT_SID: str
+    TWILIO_AUTH_TOKEN: str
+
+    # SID stands for String Identifier: SMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    # It’s a unique key that is used to identify specific resources. At Twilio, each SID has 34 digits
+    # and you can identify the type of SID and the product it’s associated with
+    # by the first two characters.
+    TWILIO_MESSAGING_SID: str = Field(
```

Comment:
So this is defined on twill's service and we have to configure it here, right? I'm not sure I got it how this is used and it's purpose. So when sending a normal SMS, this would be a phone number right? Like `+41...` or `Sunrise`?

Reply:
yes